### ISSUE TITLE
[Bug]: False Positive

### ISSUE THREAD

**Issue by kmcclain-entre** [CyberDrain/Check#132]
### Required confirmations before submitting

- [x] **I can reproduce this issue on the latest released version** of Check.
- [x] **I have searched existing issues** (both open and closed) to avoid duplicates.
- [x] I am **not** requesting general support; this is an actual bug report.

### Issue Description

www.alaskaair.com

### Extension Version

1.1.0

### Rules Version

1.1.0

### Relevant Logs / Stack Trace

```plaintext
[
  "phi_031_suspicious_query_length_combined",
  "phi_033_suspicious_event_listeners",
  "phi_004",
  "phi_012_suspicious_resources",
  "phi_006",
  "phi_010_aad_fingerprint",
  "phi_013_form_action_mismatch",
  "phi_014_devtools_blocking",
  "phi_015_code_obfuscation",
  "phi_001_enhanced",
  "phi_002",
  "phi_003",
  "phi_020_grammar_typos",
  "phi_017_microsoft_brand_abuse",
  "phi_023_css_selection_blocking",
  "phi_029_fake_dead_links",
  "phi_030_empty_tag_obfuscation"
]

Critical phishing indicators detected on non-Microsoft page: phi_010_aad_fingerprint, phi_013_form_action_mismatch, phi_001_enhanced

https[:]//www.alaskaair.com/
```

**Comment by Zacgoose** [comment]
fixed with latest version
